cumulus/scripts/bridges_rococo_westend.sh

```diff
@@ -132,10 +132,10 @@ LANE_ID="00000002"
 XCM_VERSION=3
 
 function init_ro_wnd() {
-    ensure_relayer
+    local relay_path=$(ensure_relayer)
```

Comment:
Nit: Should this be named `relayer_path` ? The same question for  the functions below.